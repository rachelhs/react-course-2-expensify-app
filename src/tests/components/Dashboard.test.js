import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashboardPage from '../../components/Dashboard';
import toJSON from 'enzyme-to-json';

test('should render Dashboard page', () => {
    const wrapper = shallow(<ExpenseDashboardPage />);
    expect(toJSON(wrapper)).toMatchSnapshot();
});