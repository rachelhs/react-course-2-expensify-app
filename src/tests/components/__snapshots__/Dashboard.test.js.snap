// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Dashboard page 1`] = `
<div>
  <Connect(ExpenseSummary) />
  <Connect(ExpenseList) />
  <Connect(ExpenseListFilters) />
</div>
`;
